### PR TITLE
Android Code Reformatting

### PR DESCRIPTION
**Pull request by shubham7109**
## Description
- Ran file-reformat on all the Kotlin and Java samples
- Code cleanup on the entire project
- Updated Kotlin samples to use 4 spaces for indentation
- Removed unused imports, resources, and fixed code style issues.

## Links and Data
https://devtopia.esri.com/runtime/common-samples/issues/3196

## What To Review
Review the changes made to the code using the Android code inspection tools

## How to Test
The sample behavior should not change in this PR, so test if the samples build and run without errors.

### CODE REVIEW

**Review by hud10837 — COMMENTED** [review]
@shubham7109 I wasn't able to get through all of them, but I noticed some formatting things that might be fixable

File: java/download-preplanned-map-area/src/main/java/com/esri/arcgisruntime/sample/downloadpreplannedmaparea/MainActivity.java [hunk 1]
```diff
@@ -265,12 +265,8 @@ private void createPreplannedAreasListView(ArcGISMap onlineMap, OfflineMapTask o
...
             mMapView.setViewpointAsync(new Viewpoint(areaOfInterest), 1.5f);
             // enable download button only for those map areas which have not been downloaded already
-            if (new File(getCacheDir() + getString(R.string.preplanned_offline_map_dir) + File.separator
-                + mSelectedPreplannedMapArea.getPortalItem().getTitle()).exists()) {
-              mDownloadButton.setEnabled(false);
-            } else {
-              mDownloadButton.setEnabled(true);
-            }
+            mDownloadButton.setEnabled(!new File(getCacheDir() + getString(R.string.preplanned_offline_map_dir) + File.separator
+                    + mSelectedPreplannedMapArea.getPortalItem().getTitle()).exists());
```
[the remark this answers is not in the document]
Comment (shubham7109) [reply java/download-preplanned-map-area/src/main/java/com/esri/arcgisruntime/sample/downloadpreplannedmaparea/MainActivity.java:269]: This is a change made automatically by running code-cleanup on the entire project. There is a good point to be made here. Should we accept such changes, where the original code is easier to read than the simplified version? @TADraeseke What would you suggest here? I could comb through and revert similar instances if we decide to keep the original version.  

Original code: 
```kotlin
            if (new File(getCacheDir() + getString(R.string.preplanned_offline_map_dir) + File.separator
                + mSelectedPreplannedMapArea.getPortalItem().getTitle()).exists()) {
              mDownloadButton.setEnabled(false);
            } else {
              mDownloadButton.setEnabled(true);
            }
```
Running code-cleanup:
```kotlin
              mDownloadButton.setEnabled(!new File(getCacheDir() + getString(R.string.preplanned_offline_map_dir) + File.separator
                      + mSelectedPreplannedMapArea.getPortalItem().getTitle()).exists());
```

**Review by TADraeseke — CHANGES_REQUESTED** [review]
Thanks @shubham7109! I imagine this was a bit of a thankless task, but good to get these things cleaned up.

I only found a few minor things below. Thanks!

**Review by TADraeseke — APPROVED** [review]
LGTM!

File: kotlin/display-map/src/main/java/com/esri/arcgisruntime/sample/displaymap/MainActivity.kt [hunk 2]
```diff
@@ -33,50 +33,30 @@ import com.esri.arcgisruntime.sample.displaymap.databinding.ActivityMainBinding
...
-    mapView.dispose()
-    super.onDestroy()
-  }
-  //[DocRef: END]
-}
+        //[DocRef: Name=Set map-Android, Category=Get started, Topic=Develop your first map app with Kotlin]
+        // set the map to be displayed in the layout's MapView
+        mapView.map = map
+        //[DocRef: END]
+    }
```
Comment (saratchandrakarumuri) [inline kotlin/display-map/src/main/java/com/esri/arcgisruntime/sample/displaymap/MainActivity.kt:62]: missing closing `}` after this... This is causing a build error

**Review by saratchandrakarumuri — CHANGES_REQUESTED** [review]
Except for that one issue everything is looking good

File: kotlin/display-map/src/main/java/com/esri/arcgisruntime/sample/displaymap/MainActivity.kt (hunk 2 above)
Comment (shubham7109) [reply kotlin/display-map/src/main/java/com/esri/arcgisruntime/sample/displaymap/MainActivity.kt:62]: Missed this while resolving merge conflicts. Good catch 👍

**Review by saratchandrakarumuri — APPROVED** [review]
Looks good

**Pull request merged**